plib/library/Helper.php

```diff
@@ -561,12 +561,14 @@ public static function getDomains()
                 continue;
             }
 
+            $dns_ip_address = $domain->data->gen_info->dns_ip_address ?? [];
```

Comment:
It will be enough to make a forced cast:
`(array)$domain->data->gen_info->dns_ip_address`